.mypy.ini

```diff
@@ -0,0 +1,25 @@
+[mypy]
```

Comment:
The basic idea here is to completely disallow all dynamic typing in these modules. This ensures mypyc has all the information needed to provide the best possible performance improvements.